src/ape/_cli.py

```diff
@@ -72,15 +72,22 @@ def _suggest_cmd(usage_error):
 
     @property
     def commands(self) -> Dict:
+        group_name = "ape_cli_subcommands"
```

Comment:
This part shouldn't be in the diff, may need to catch your origin main up or catch this branch up or both, not sure!